CHANGELOG.md

```diff
@@ -4,138 +4,138 @@
-- 3ff82d0: **template:** seek-jobs/gantry v1.3.0
+- **template:** pino-pretty ^5.0.0 ([#441](https://github.com/seek-oss/skuba/pull/441))
+- **template:** seek-jobs/gantry v1.3.0 ([#452](https://github.com/seek-oss/skuba/pull/452))
 
 ## 3.14.0
 
 ### Minor Changes
 
-- 88ad97f: **deps:** Prettier 2.3
+- **deps:** Prettier 2.3 ([#434](https://github.com/seek-oss/skuba/pull/434))
 
   This release may require reformatting of your code. If your lint step is failing in CI, run your format command locally then push the resulting changes.
 
   ```shell
   yarn format
   ```
 
-- 88ad97f: **deps:** TypeScript 4.3
+- **deps:** TypeScript 4.3 ([#434](https://github.com/seek-oss/skuba/pull/434))
 
   This major release includes breaking changes. See the [announcement](https://devblogs.microsoft.com/typescript/announcing-typescript-4-3/) for more information.
 
   `skuba format` and `skuba lint` will error on new TypeScript 4.3 syntax that are not yet supported by Prettier.
 
-- 28c62d4: **deps:** Jest 27
+- **deps:** Jest 27 ([#433](https://github.com/seek-oss/skuba/pull/433))
 
   This major release includes breaking changes. See the Jest 27 [blog post](https://jestjs.io/blog/2021/05/25/jest-27) and [changelog](https://github.com/facebook/jest/blob/v27.0.3/CHANGELOG.md) for more information.
 
 ### Patch Changes
 
-- 7ec71e8: **Jest.mergePreset:** Type `snapshotSerializers` option
-- 1742096: **template:** Banish `typeof undefined` syntax
-- 8842f92: **template/lambda-sqs-worker-cdk:** Always build before deploy
+- **Jest.mergePreset:** Type `snapshotSerializers` option ([#436](https://github.com/seek-oss/skuba/pull/436))
+- **template:** Banish `typeof undefined` syntax ([#429](https://github.com/seek-oss/skuba/pull/429))
+- **template/lambda-sqs-worker-cdk:** Always build before deploy ([#428](https://github.com/seek-oss/skuba/pull/428))
 
   This prevents stale compiled code from being cached and deployed from ECR.
 
-- 1b72b4b: **template/koa-rest-api:** Log returned error responses
-- 50792c8: **template:** Prune `devDependencies` instead of installing twice in Docker
+- **template/koa-rest-api:** Log returned error responses ([#430](https://github.com/seek-oss/skuba/pull/430))
+- **template:** Prune `devDependencies` instead of installing twice in Docker ([#435](https://github.com/seek-oss/skuba/pull/435))
 
   The template-bundled Dockerfiles would previously run `yarn install` twice to build a separate stage for production `dependencies` only. These have been updated to correctly share the Yarn cache across stages and to use `yarn install --production` to perform offline pruning.
 
-- d4ef636: **deps:** fs-extra ^10.0.0
+- **deps:** fs-extra ^10.0.0 ([#424](https://github.com/seek-oss/skuba/pull/424))
 
 ## 3.13.1
 
 ### Patch Changes
 
-- 4ecd622: **template/\*-npm-package:** Add `yarn commit` script
-- 94f3c4d: **template/lambda-sqs-worker-cdk:** Trim CDK deployment output
-- bc0d95d: **template:** @types/node ^15.0.0
```

Comment:
Fun tidbit: the script found that this entry was mistagged and should've been https://github.com/seek-oss/skuba/commit/27d5ea3e734ea9ca56771ea9eac69a44299ef67e.